internal/driver/cli.go

```diff
@@ -34,6 +34,9 @@ type source struct {
 	Timeout      int
 	Symbolize    string
 	HTTPHostport string
+
+	// Comment will be appended to any output format that supports comments.
```

Comment:
Can you list these output formats?

Reply:
good question. i'm not sure I can, and i dont know if thats worthwhile. the comment gets appended to the profile before it gets processed by any of the other commands, and its up to them whether they do anything with comments. i assume most of them ignore comments, except for proto, raw and comments.

perhaps it would be better to say something "append comment to profile before processing to output format?"